Automattic/woocommerce-payments
# Update to fix JS error when 100% off coupon is applied

Fixes #1186 

#### Changes proposed in this Pull Request

*  Update for a JS error which occurs when a 100% off coupon is applied. This error is caused because of an event which triggers when checkout is updated and part of it tries to check the `.length` of the `children()` of `$('#wcpay-card-element')`. If there is no fee to be paid, this element disappears from the page which is why this error was occurring.

#### Testing instructions

* Create a 100% off coupon in Marketing -> Coupons.
* Apply the coupon during checkout of a product.
* Verify that on `trunk` a JavaScript error occurs in the console.
* Verify that on this branch, the error no longer appears.

-------------------
